src/scss/themes/satellite.scss

```diff
@@ -590,6 +590,17 @@ $break-medium: 767px;
   font-style: normal;
 }
 
+/**
+ * ReverseHighlighting and ReverseSnippeting
+ */
+
+.ais-ReverseHighlight-highlighted,
+.ais-ReverseSnippet-highlighted {
```

Comment:
We need to add `font-style: normal` because `em` (which is the default highlighting tag that Algolia uses) has italic style in most browsers.

Reply:
Got it! Thanks